cli/README.md

```diff
@@ -0,0 +1,206 @@
+## divviup - the Divvi Up command line tool
+
+`divviup` is a command line (CLI) tool for doing all basic operations on both the Divvi Up API and Distributed Aggregation Protocol (DAP) API endpoints. See `divviup --help` for details on all of the commands.
+
+### Command Line Tutorial
+
+First, set an environment variable for the Divvi Up account and an API token. The initial token will be created when an account is created.
+
+```
+export TOKEN=YOUR_TOKEN_HERE
```

Comment:
```suggestion
export DIVVIUP_TOKEN=YOUR_TOKEN_HERE
```

If we export to this variable, we won't need to supply `--token` to any of the subsequent commands.

Reply:
We also need to `export DIVVIUP_ACCOUNT_ID={your account ID here}` for account-specific commands to work.

Reply:
I think you only need `DIVVIUP_ACCOUNT_ID` if the token has administrator privileges, otherwise the CLI can look up the account ID it is bound to.

Reply:
I agree with this suggestion, and I think it still needs to be made (possibly in-flight?)

Reply:
Indeed, the `DIVVIUP_TOKEN` suggestion is still applicable.